src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java

```diff
@@ -101,6 +102,7 @@ private ConfigurationRepository(Settings settings, final Path configPath, Thread
         this.clusterService = clusterService;
         this.auditLog = auditLog;
         this.configurationChangedListener = new ArrayList<>();
+        this.acceptInvalid = settings.getAsBoolean(ConfigConstants.OPENDISTRO_SECURITY_UNSUPPORTED_ACCEPT_INVALID_CONFIG, false);
```

Comment:
Curious why we have this property - Whats the use case of starting up nodes always accepting invalid config? 
If it's just for loading during migration API call, we can just overload the function to do it?

Reply:
accepting invalid config is configuration parameter you can set in elasticsearch.yml 

You can configure if you want to accept invalid/non-supported 6.x configuration and still migrate remaining valid configuration to 7.x format and still start node. 

7.x can still read 6.x configuration and work without any problem. 

If you migrate configuration with accept invalid configuration to true, you need that flag everytime when node boots up to ignore invalid configuration present in index.